docs/alps/storage.md

```diff
@@ -16,20 +18,73 @@ These separate clusters are on the same Slingshot 11 network as the Alps.
 | IOPs         | 1.5M                   | 8.6M read, 24M write   | 200k read, 768k write |
 | file create/s| 374k                   | 214k                   | 97k                 |
 
+
+!!! todo
+    Information about LUSTER. Meta data servers, etc.
+
+    * how many meta data servcers on capstor and iopstor
+    * how these are distributed between store/scratch
+
+    Also discuss how capstor and iopstor are used to provide both scratch / store / other file systems
+
 [](){#ref-alps-capstor}
-## capstor
+## Capstor
 
 Capstor is the largest file system, for storing large amounts of input and output data.
-It is used to provide SCRATCH and STORE for different clusters - the precise details are platform-specific.
+It is used to provide [scratch][ref-storage-scratch] and [store][ref-storage-store].
+
+!!! todo "add information about meta data services, and their distribution over scratch and store"
+
+[](){#ref-alps-capstor-scratch}
+### Scratch
+
+All users on Alps get their own scratch path on Alps, `/capstor/scratch/cscs/$USER`.
+
+[](){#ref-alps-capstor-store}
+### Store
+
+The Store mount point on Capstor provides stable storage with [backups][ref-storage-backups] and no [cleaning policy][ref-storage-cleanup].
```

Comment:
Reading the rest of the document, I think capitalized "Store" is the preferred spelling?

Reply:
Proper nouns it is.

I have updated capitalization to the following:

Store, Scratch, Home, Capstor, Iopsstor, VAST.